src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -1913,6 +1913,15 @@ DatabaseShardImp::callForLedgerSQL(
     std::lock_guard lock(mutex_);
     auto shardIndex = seqToShardIndex(ledgerSeq);
 
+    return doCallForLedgerSQL(shardIndex, callback);
```

Comment:
The lock can come after the declaration of shardIndex. The local can also be const.

eg. auto const shardIndex{seqToShardIndex(ledgerSeq)};

Reply:
Done